routes/user.go

```diff
@@ -3318,6 +3319,56 @@ func (fes *APIServer) GetTransactionSpendingLimitHexString(ww http.ResponseWrite
 	}
 }
 
+type GetAccessBytesRequest struct {
+	DerivedPublicKeyBase58Check string
+	ExpirationBlock             uint64
+	TransactionSpendingLimit    TransactionSpendingLimitResponse
+}
+
+type GetAccessBytesResponse struct {
+	SpendingLimitHex string
```

Comment:
we should probably keep this naming consistent - I think we call it `TransactionSpendingLimitHex` in other places, but not  a huge deal either way.